_posts/2021-12-15-hhvm-4.140.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: "HHVM 4.140"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.140 is released! This release marks the end of support for 4.134; HHVM
+4.135&ndash;4.139 remain supported, as do the 4.102 and 4.128 LTS releases.
+
```

Comment:
Might be worth also mentioning that this will be the last scheduled release of the year - https://hhvm.com/blog/2021/12/09/hhvm-4.139.html